Added latest workset note, statuses and "copy table"-button to the worksets table

sofiahaglund opened the pull request:
WIth a lot of help from Anand I managed to add the latest workset note to the worksets table!  

Since there are so many columns in this table, I added the note to the far right of the "default view" of the screen, since I guess it's more valuable to see the latest workset note during the production meetings than the number of samples passed/failed/pending? To see these columns one can simply scroll to the right. However I'm open to suggestions, so if you want I could add the note to the far right (not visible in the "default view") as well.

It's on stage, since we're using tools-dev one has to click "all" (there is only one workset with a note :-))...

aanil commented on status/worksets.py:
```diff
@@ -20,12 +20,12 @@ class WorksetsDataHandler(SafeHandler):
             result[row.key].pop("_id", None)
             result[row.key].pop("_rev", None)
-        self.write(json.dumps(result))
+        self.write(json.dumps(result,result2))
```
What does the result2 return?

sofiahaglund replied:
oooops, forgot to remove!

aanil commented on run_dir/design/worksets.html:
```diff
@@ -73,18 +73,39 @@ <h2 id="{{ worksets }}" class="worksets_page_heading"> <small>Worksets from the
         </div>
       </div>
-
+      <script type="text/javascript">
```
Is make_markdown called on the panel body twice, once here and once in load_table() in worksets.js?

sofiahaglund replied:
It is! Hmmm....I only added it to the HTML-file because when I inspected in the browser, the markdown class wasn't applied to the workset note as it was to the running note.....I can check again!

sofiahaglund replied:
Yeah, it's not showing the markdown:
<img width="530" alt="Skärmavbild 2019-12-06 kl  13 43 34" src="https://user-images.githubusercontent.com/37932118/70324012-a42a0180-182e-11ea-90c8-92468268d3e9.png">
When I added the extra lines of code in the HTML:
<img width="528" alt="Skärmavbild 2019-12-06 kl  13 40 30" src="https://user-images.githubusercontent.com/37932118/70324034-b73cd180-182e-11ea-99ad-91dc229256ec.png">

sofiahaglund commented:
I put the latest workset notes back to the far right, added the only two statuses that are in the database, "open" and "closed", to be used for now, and changed the color of the workset name links to not clash with that of the project links :-)

sofiahaglund commented:
aaaand I removed the classes for the status coloring and replaced these with the built-in bootstrap classes...

sofiahaglund commented:
added a "copy table to clipboard"-button while at it :-)
